apps/token-e2e/src/integration/flow/token-association-flow.cy.js

```diff
@@ -0,0 +1,118 @@
+/// <reference types="cypress" />
+import navigation from '../../locators/navigation.locators';
+import staking from '../../locators/staking.locators';
+import wallet from '../../locators/wallet.locators';
+import '../../support/staking.functions';
+import '../../support/vega-wallet.functions';
+import '../../support/eth-wallet.functions';
+import '../../support/wallet-teardown.functions';
+
+const vegaWalletPublicKeyShort = Cypress.env('vegaWalletPublicKeyShort');
+
+context('Staking Tab - with eth and vega wallets connected', function () {
```

Comment:
Suggestion (non-blocking): Description/context of flows tests should describe functionality that is tested rather than page/tab. Same for staking-flow.cy.js

Reply:
will address in a later release